python_modules/dagster/dagster/_core/definitions/declarative_scheduling/operands/parent_newer_condition.py

```diff
@@ -0,0 +1,106 @@
+from typing import AbstractSet
+
+from dagster._core.asset_graph_view.asset_graph_view import AssetSlice
+from dagster._core.definitions.declarative_scheduling.scheduling_condition import (
+    SchedulingCondition,
+    SchedulingResult,
+)
+from dagster._core.definitions.declarative_scheduling.scheduling_context import SchedulingContext
+from dagster._core.definitions.events import AssetKeyPartitionKey
+from dagster._serdes.serdes import whitelist_for_serdes
+
+
+@whitelist_for_serdes
+class ParentNewerCondition(SchedulingCondition):
+    @property
+    def description(self) -> str:
+        return "At least one parent has been updated more recently than the candidate."
+
+    def compute_newer_parents(
+        self, context: SchedulingContext, candidate: AssetKeyPartitionKey
+    ) -> AbstractSet[AssetKeyPartitionKey]:
+        """Returns the set of parent asset partitions that are newer than the candidate."""
+        # TODO: replace this implementation with one backed by the StaleStatusResolver
+        candidate_slice = context.asset_graph_view.get_asset_slice_from_asset_partitions(
+            {candidate}
+        )
+        parent_slices = candidate_slice.compute_parent_slices()
+        all_parent_asset_partitions = {
+            ap
+            for parent_slice in parent_slices.values()
+            for ap in parent_slice.compute_asset_partitions()
+        }
+        return (
+            context.legacy_context.instance_queryer.get_parent_asset_partitions_updated_after_child(
+                candidate,
+                all_parent_asset_partitions,
+                False,
```

Comment:
keyword arg on the boolean please. 

https://github.com/dagster-io/internal/discussions/9682 :-)